css-ui-4/Overview.bs

```diff
@@ -1081,26 +1081,57 @@ Cursor of the canvas</h5>
 <h3 id="insertion-caret">
 Insertion caret</h3>
 
+	The <dfn lt="insertion caret|caret">insertion caret</dfn> (or “caret” for short)
+	is a visible indicator of the insertion point in an element that [=accepts input=],
+	where text (and potentially other content) can be inserted by the user.
+	The 'caret-color', 'caret-animation', and 'caret-shape' properties
+	give the author some amount of control over the appearance of the [=caret=].
+
+	Elements are considered to <dfn>accept input</dfn>
```

Comment:
In order to respect https://github.com/w3c/csswg-drafts/issues/5303#issuecomment-714593347, text may also be considered to accept input and then these properties should apply to it.

```html
<!DOCTYPE html>
<div contenteditable>
  <span style="display: contents; caret-color: magenta">aaaaaaa</span>
</div>
```